src/main/java/database/Database.java

```diff
@@ -37,6 +42,35 @@ private Database() {
             String jdbcUrl = "jdbc:postgresql://" + hostname + ":" + port + "/" + dbName + "?user=" + userName + "&password=" + password;
             this.connection = DriverManager.getConnection(jdbcUrl);
             System.out.println("Database connected");
+
+            //Creating tables
+            Statement users = this.connection.createStatement();
+            String sqlUser = "CREATE TABLE USERS " +
+                         "(ID INT PRIMARY KEY   NOT NULL," +
+                         " NAME         VARCHAR(50)  NOT NULL, " +
```

Comment:
... wow i was thinking name was their name and username is what they sign in with, but we don't really have a name. my bad haha